databroker/tests/test_v2/test_jsonl.py

```diff
@@ -7,9 +7,11 @@
 import tempfile
 import time
 import types
+import yaml
```

Comment:
No longer used. (My fault for leaving this behind in my test refactor.)

Reply:
updated